src/main/java/com/bugsnag/android/Client.java

```diff
@@ -287,22 +287,22 @@ public void setUserName(String name) {
     /**
      * Add a "before notify" callback, to execute code before every
      * notification to Bugsnag.
-     *
+     * <p/>
      * You can use this to add or modify information attached to an error
      * before it is sent to your dashboard. You can also return
      * <code>false</code> from any callback to halt execution.
-     *
+     * <p/>
      * For example:
+     * <p/>
+     * client.beforeNotify(new BeforeNotify() {
+     * public boolean run(Error error) {
+     * error.setSeverity(Severity.INFO);
+     * return true;
+     * }
+     * })
```

Comment:
This formatting probably needs corrected to the original version. Maybe Android Studio would like a `<code>` block better.